package.json

```diff
@@ -7,8 +7,11 @@
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/integration/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress run",
+    "combine-reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+  "generate-report": "marge cypress/reports/report.json -f report -o cypress/reports"
```

Comment:
fix the indentation

Reply:
Done

Reply:
I just finished the indentation, :)  I will work on the report and send the screenshot.

Reply:
Yeah, please remember to use the npm pre and post hooks to run scripts before and after tests

Reply:
![ReporteMochawesomeAnto](https://user-images.githubusercontent.com/44952460/115126796-74008000-9f97-11eb-89eb-bd8f911afd37.jpg)


Finally!

Reply:
I had some problems, the steps on de workshop say to install mochawesome npm install mocha --save-dev, but to me, worked whit npm install mochawesome --save-dev. 

I found the cypress documentation and help to me a lot. 

https://docs.cypress.io/guides/tooling/reporters#Examples
https://www.yld.io/blog/using-npm-pre-and-post-hooks/

Also, the terminal (I worked with the windows terminal) doesn't recognize the RM command, which confused me a lot, but then I understood that command is for Linux. :S

Reply:
Really good job anto!

Reply:
:) Thank you, and thank you to you for teaching us this. 

I done with the changes!